### ISSUE TITLE
No image at coding challenge "Know When Alt Text Should be Left Blank"

### ISSUE THREAD

**Issue by sondregi** [freeCodeCamp/freeCodeCamp#41131]
**Describe your problem and how to reproduce it:**
No image at coding challenge 

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/responsive-web-design/applied-accessibility/know-when-alt-text-should-be-left-blank

**Tell us about your browser and operating system:**
* Browser Name: Brave
* Browser Version: V1.20.103
* Operating System:  MacOS Big Sur v11.1

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
<img width="1419" alt="Skjermbilde 2021-02-15 kl  16 25 42" src="https://user-images.githubusercontent.com/18233699/107965175-7bf96f00-6faa-11eb-8485-f417698e574c.png">

**Comment by RandellDawson** [comment]
@sondregi The image not loading is by design.  See the following sentence in the challenge description:
>This helps in cases where the image fails to load or can't be seen by a user. 

When an image does not load (for any reason), you still want to have an `alt` text to describe the image.

I am closing this as a non-issue.  If you feel this needs to be discusses further, please request it to be reopened.

**Comment by sondregi** [comment]
Haha, I'm sorry. I need to read full description 😂
